### PR TITLE
feat:implemented button component

### PR DESCRIPTION
**Pull request by Frank1290**

### CODE REVIEW

File: resources/js/components/Button/ButtonComponent.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "React";
+import PropTypes from "prop-types";
+
+const ButtonComponent = ({ name, sty }) => {
```
[the remark this answers is not in the document]
Comment (Frank1290) [reply resources/js/components/Button/ButtonComponent.js]: Done

File: resources/js/components/WelcomePage/WelcomeComponent.js [hunk 2]
```diff
@@ -1,18 +1,26 @@
-import React from "react";
+import React, { Component } from "react";
```
[the remark this answers is not in the document]
Comment (Frank1290) [reply resources/js/components/WelcomePage/WelcomeComponent.js]: Done

File: resources/js/components/Button/ButtonComponent.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    console.log(name, sty);
```
[the remark this answers is not in the document]
Comment (Frank1290) [reply resources/js/components/Button/ButtonComponent.js]: Done

File: resources/js/components/WelcomePage/WelcomeComponent.js [hunk 4]
```diff
@@ -1,18 +1,27 @@
...
+    return (
+        <div>
+            <h2 className="heading">WELCOME TO JOB PORTAL</h2>
+            <div className="button-pos">
+                <Link to="/postJob">
+                    <ButtonComponent
+                        name={button.post}
+                        classNames={button.style}
+                    />
+                    <ButtonComponent name={button.search} />
```
Comment (devechha21ramekar) [inline resources/js/components/WelcomePage/WelcomeComponent.js:20]: can you move 2nd button outside link

File: resources/js/components/WelcomePage/WelcomeComponent.js [hunk 5]
```diff
@@ -1,18 +1,26 @@
...
+    return (
+        <div>
+            <h2 className="heading">WELCOME TO JOB PORTAL</h2>
+            <div className="button-pos">
+                <Link to="/postJob">
+                    <ButtonComponent name={button.post} sty={button.style} />
+                    <ButtonComponent name={button.search} />
+                </Link>
+
+                {/* <button className="welcome-button">Search Job</button> */}
```
[the remark this answers is not in the document]
Comment (Frank1290) [reply resources/js/components/WelcomePage/WelcomeComponent.js]: Done

File: resources/js/components/WelcomePage/WelcomeComponent.js (hunk 4 above)
Comment (Frank1290) [reply resources/js/components/WelcomePage/WelcomeComponent.js]: done

File: resources/js/components/WelcomePage/WelcomeComponent.js [hunk 6]
```diff
@@ -1,18 +1,27 @@
...
-    <div>
-        <h2 className="heading">WELCOME TO JOB PORTAL</h2>
-        <div className="button-pos">
-            <Link to="/postJob">
-                <button className="welcome-button text-white btn-primary">
-                    ADD Job
-                </button>
-            </Link>
+import ButtonComponent from "../Button/ButtonComponent";
+const button = {
```
Comment (devechha21ramekar) [inline resources/js/components/WelcomePage/WelcomeComponent.js:4]: button is a HTML keyword . can you please change it to buttonObj

Comment (devechha21ramekar) [reply resources/js/components/WelcomePage/WelcomeComponent.js:4]: last one plz

Comment (Frank1290) [reply resources/js/components/WelcomePage/WelcomeComponent.js]: Done!!

**Pull request merged**